product-service/handler.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const data = require('./cats');
+
+const headers = {
+  'Content-Type': 'application/json',
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept',
+  'Access-Control-Allow-Methods': 'POST, GET, OPTIONS, PATCH, PUT',
+}
+
+module.exports.getProductsList = async event => {
```

Comment:
what is the purpose of using `async` keyword? Thera are no async operations in handler's body

Reply:
A cheap way not to create a Promise explicitly